libafl/src/executors/mod.rs

```diff
@@ -15,9 +15,9 @@ pub mod timeout;
 #[cfg(any(unix, feature = "std"))]
 pub use timeout::TimeoutExecutor;
 
-#[cfg(all(feature = "std", feature = "fork", unix))]
+#[cfg(all(feature = "regex", feature = "fork", unix))]
```

Comment:
I don't understand why we need regex for the forkserver(?)

Reply:
i think it should be std here

Reply:
ah, so regex incldues std

Reply:
The forkserver uses the `AsanBacktraceObserver`, which itself needs `regex`:

https://github.com/AFLplusplus/LibAFL/blob/31357aa7e2f1aba78d99db6cd3ae3243202ddb25/libafl/src/executors/forkserver.rs#L39

Reply:
ah ok!

Reply:
I think we should just not support `AsanBacktraceObserver` without `regex`, but the forkserver exeutor should still work - it's a special case anyway

Reply:
As in: just wrap these two checks in a cfg:
https://github.com/AFLplusplus/LibAFL/blob/main/libafl/src/executors/forkserver.rs#L1125-L1136

Reply:
Done!